pandas/tests/arrays/test_array.py

```diff
@@ -0,0 +1,109 @@
+import decimal
+
+import numpy as np
+import pytest
```

Comment:
shouldn't this be in tests/extensions? and part of the extension base infrastructure?

Reply:
This is testing `pd.array` itself, not something that's part of the interface.

I did add a couple tests in `base/constructors.py` to check that `pd.array` is idempotent for EAs.